src/HttpHeader.h

```diff
@@ -179,6 +182,7 @@ class HttpHeader
     static bool Isolate(const char **parse_start, size_t l, const char **blk_start, const char **blk_end);
     bool skipUpdateHeader(const Http::HdrType id) const;
     void updateWarnings();
+    size_t maxLen() const; ///< the maximum packed length allowed, zero means 'not restricted'
```

Comment:
Stale new comment
```suggestion
    size_t maxPackedSize() const; ///< the maximum number of packed bytes allowed
```

Reply:
This is not needed since I removed maxLen().